.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build and Publish JAR Packages
+run-name: ${{ inputs.release_type == 'Snapshot' && 'Publish pre-release' || format('Release {0}', inputs.release_type)}} Package by @${{ github.actor }}
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: The type of release
+        options:
+        - Major
+        - Minor
+        - Patch
+      publish_vulnerabilities:
+        type: string
+        default: true
+      dotnet_version:
+        type: string
+        default: 5.0
```

Comment:
is this just for dot net version for running to build the binary or does this correspond to some setting in the c# SDK repo?

Reply:
or i just would like to know when should we modify this?

Reply:
At the moment this is used to build/package/test dotnet.

When we bump the version in the code, we can change this in the action interface.